src/js/media-chrome-button.ts

```diff
@@ -61,14 +70,36 @@ template.innerHTML = /*html*/ `
     max-height: 100%;
     min-width: 100%;
   }
+
+  media-tooltip {
+    opacity: 0;
+    transition: opacity .3s;
```

Comment:
can be an iteration but I'd probably like to change the:
- transition delay
- transition duration
- maybe the transform of the tooltip 
- maybe the easing of the transition